### ISSUE TITLE
Git tracking issue with cljs-out directory

### ISSUE THREAD

**Issue by azureus89** [224-1/psilib-html5-theremin#12]
We shouldnt be sending the cljs output files back and forth, there are 340 of them, and its slowing things down. Need to find a way to untrack it and a simple system to keep output directory outside of git. .gitignore doesnt seem to be working because git add * seems to over-ride it ... further people may make a mistake and suddenly track the files on magit, so its not a robust solution to only depend on .gitignore perhaps.

**Comment by ParikshithMohan** [comment]
So staging only core.cljs is fine no?

**Comment by azureus89** [comment]
No, the problem is at the tracking layer, not the staging layer. We have to stop git from tracking the output folder henceforth AND apply one small fix so that it works retrospectively also, otherwise it will cause merge issues.
